src/components/notes/saving-indicator.spec.js

```diff
@@ -33,12 +49,27 @@ function itMatchSnapshot(collabProvider) {
         }
       }
     )
-    expect(component.html()).toMatchSnapshot()
+    expect(normalizeMUI(component.html())).toMatchSnapshot()
   })
 }
 
 describe('SavingIndicator', () => {
-  afterEach(() => jest.resetAllMocks())
+  afterEach(() => {
+    jest.resetAllMocks()
+  })
+  beforeEach(() => {
+    useBreakpoints.mockImplementation(() => ({
+      isMobile: false
+    }))
+    global.document.createRange.mockImplementation(() => ({
```

Comment:
Nit: A comment stating why document.createRange could be useful (at first I thought it was the same as lodash's range function, I forgot about this name for selections).